### PR TITLE
fix(value): values correctly set programmatically #75

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #75

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jordanjones243 — APPROVED** [review]

File: src/auro-combobox.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ class AuroCombobox extends LitElement {
     this.noFilter = false;
     this.value = null;
     this.optionSelected = null;
+    this.error = false;
```
Comment (blackfalcon) [inline src/auro-combobox.js:48]: Nit: Why was the default set to `false`? The only reference I see here is to set `this.error='true'`

I am curious as to what value pre-setting this to `false` in the constructor brings versus leaving it as `undefined`? Is this a Svelte issue?

**Review by blackfalcon — DISMISSED** [review]
One question, otherwise approved.

File: src/auro-combobox.js (hunk 1 above)
Comment (jason-capsule42) [reply src/auro-combobox.js]: Wasn't actually necessary. I removed it.

**Pull request merged**